src/snowplow/tracker.lua

```diff
@@ -55,14 +66,10 @@ function tracker.new_tracker(emitter, encode_base64)
   return trck
 end
 
--- --------------------------------------------------------------
--- Private methods
-
--- Tracks any given SnowPlow event, by sending the specific event_pairs to the SnowPlow collector.
--- @param: self table: The Tracker instance
--- @param pb table: A partially populated payload_builder closure. We will finish populating it in this method, then
--- build() it
--- @return boolean, string: Whether event was successfully collected; and the reason for failure if not
+--- Tracks any given SnowPlow event, by sending the specific event_pairs to the Snowplow collector.
```

Comment:
The "SnowPlow" capitalisation 🙈

Reply:
A relic of the before times, I've updated it to 'Snowplow' 😉